spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -273,7 +299,7 @@ icaPacketData = InterchainAccountPacketData{
 }
 
 // Sends the message to the host chain, where it will eventually be executed 
-SendTx(ownerAddress, portID, data, timeout)
+SendTx(ownerAddress, connectionId, portID, data, timeout)
```

Comment:
@AdityaSripal @colin-axner What is the `connectionId` here exactly?

Reply:
The `connectionId` here identifies the ICS3 connection that the channel is built on top of. ICA only allows one ICA channel per connection to allow for reopening logic